tests/QMLEngine/qml/ImportLocalToFile/ModuleFail.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.0
+
+Item {
+    ModuleFailSomeComponent {
+        WebSocket {}
+    }
+    WebSocket {}
```

Comment:
I believe I put that there because the current behaviour would have made the first WebSocket valid, while the old behaviour (at least what the documentation states) is that all imports in all files are global. So the second WebSocket was to make sure we don't revert back to that... But now that I'm typing it out, the inner one would protect against that anyway... So I'm good to remove it.